src/main.js

```diff
@@ -96,8 +96,8 @@ define(["storage", "websockets", "account/morph-props"], function(storage, ws, P
     appendParagraph(" Click the \"Log setStatsEvent\" button to fire setStatsEvent.");
```

Comment:
Is it worth mentioning that you need to open the dev tools to see the console.log output when you've clicked the button?

Reply:
Good point. Added.